contrib/lazydocker/template.py

```diff
@@ -0,0 +1,15 @@
+pkgname = "lazydocker"
+pkgver = "0.23.3"
+pkgrel = 0
+build_style = "go"
+hostmakedepends = ["go"]
+pkgdesc = "Lazier way to manage docker"
```

Comment:
that's an undescriptive pkgdesc if i've ever seen one

Reply:
that's just the repo description though...

Reply:
there is no rule or precedent saying pkgdesc should be the same as upstream repo description

Reply:
i wouldn't go so far as to argue from precedants but fwiw a pkgdesc acts simply as a tertiary data point to quickly confirm wether the package one is inspecting from the package manager is the same as the one in mind alongside the name, and version with the url pointing to the homepage if available or docs or upstream repo.

All to say that pkgdesc that get too creative might harm that purpose as i understand it at least, but i digress.